### PR TITLE
Remove RSS feed

### PR DESCRIPTION
**Pull request by ericholscher**
----
📚 Documentation preview 📚: https://read-the-docs-website--254.com.readthedocs.build/

### CODE REVIEW

**Comment by agjohnson** [comment]
I mentioned in Front we should fix the site URL, we sort of broke that at some point by making it a relative URL. This affects feed URLs, but also other links in HTML. Ethicalads uses this:

https://github.com/readthedocs/ethicalads.io/blob/main/publishconf.py#L13-L19

We should use something similar in pelicanconf.py, or go back to using publishconf.py.

**Comment by ericholscher** [comment]
Yea, that looks like a better solution 💯

**Review by agjohnson — APPROVED** [review]

File: pelicanconf.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-SITEURL = ''
 RELATIVE_URLS = True
 
+if os.environ.get("READTHEDOCS_VERSION_TYPE") == "external":
+    # Make all URLs "domainless" in the RTD PR preview
+    # https://docs.readthedocs.io/en/stable/reference/environment-variables.html
+    SITEURL = ""
+else:
+    # This setting is needed to make the RSS/Atom feeds generate correctly
+    SITEURL = "https://https://about.readthedocs.com"
```
Comment (humitos) [inline pelicanconf.py:15]: Is the double "https://” correct here?

Comment (agjohnson) [reply pelicanconf.py:15]: It's for extra security.

Comment (agjohnson) [reply pelicanconf.py:15]: I put up a fix at #257 :smile:

**Pull request merged**